app/src/main/java/uk/gov/onelogin/features/domain/SetFeatureFlags.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.onelogin.features.domain
+
+import android.util.Log
+import javax.inject.Inject
+import uk.gov.android.features.FeatureFlags
+import uk.gov.android.features.InMemoryFeatureFlags
+import uk.gov.onelogin.appinfo.apicall.domain.model.AppInfoData
+import uk.gov.onelogin.appinfo.source.domain.model.AppInfoLocalState
+import uk.gov.onelogin.appinfo.source.domain.source.AppInfoLocalSource
+import uk.gov.onelogin.features.AppCheckFeatureFlag
+
+fun interface SetFeatureFlags {
+    fun fromAppInfo()
+}
+
+class SetFeatureFlagsImpl @Inject constructor(
```

Comment:
Shouldn't all of this be hidden behind AppInfoService, given that AppInforService is currently acting as controller for the local and remote states, and this is simply another state to control? Seems odd to bring everything to the ViewModel to just sort of send it back to the service level.

Reply:
Yeah, that's fair, will move it up into the service controller